lib/pages/movie_menu.dart

```diff
@@ -0,0 +1,68 @@
+import 'package:flutter/material.dart';
+import 'package:movie_card/widgets/menu/header.dart';
+import 'package:movie_card/shared/ui_consts.dart';
+import 'package:movie_card/shared/custom_card.dart';
+import 'package:movie_card/widgets/menu/menu_movie_details.dart';
+
+class MovieMenu extends StatefulWidget {
+  const MovieMenu({super.key});
+
+  @override
+  State<MovieMenu> createState() => _MovieMenuState();
+}
+
+class _MovieMenuState extends State<MovieMenu> {
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: COLORS.onPrimaryContainer,
+      body: Container(
+        decoration: const BoxDecoration(
+          image: DecorationImage(
+            image: AssetImage('assets/Menu_Background.png'),
+            fit: BoxFit.cover,
+          ),
+        ),
+        child: SingleChildScrollView(
+          child: Column(
+            children: [
+              const Header(),
+              Text(
+                "Lo último que viste",
```

Comment:
Use constants for this and the rest of hard-coded strings (or values in general)